rbac/management/role/serializer.py

```diff
@@ -49,6 +49,18 @@ def validate_attributeFilter(self, value):
             message = f"attributeFilter operation must be one of {ALLOWED_OPERATIONS}"
             error = {key: [_(message)]}
             raise serializers.ValidationError(error)
+        else:
+            values = value.get("value")
+            if type(values) is str and op == "in":
+                key = "format"
+                message = "attributeFilter operation 'in' expects a List value"
+                error = {key: [_(message)]}
+                raise serializers.ValidationError(error)
+            elif type(values) is list and op == "equals":
+                key = "format"
+                message = "attributeFilter operation 'equals' expects a String value"
+                error = {key: [_(message)]}
+                raise serializers.ValidationError(error)
```

Comment:
LGTM!
Nit: Instead of raising this in several places, can we use `if error, raise serializers.ValidationError(error)` after all these checks?